pkgs/development/libraries/libftdi/default.nix

```diff
@@ -27,6 +27,10 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ libusb-compat-0_1 ];
 
+  preConfigure = ''
+    export PATH="${lib.getDev libusb-compat-0_1}/bin:$PATH"
+  '';
+
   configureFlags = lib.optional (!stdenv.hostPlatform.isDarwin) "--with-async-mode";
```

Comment:
```suggestion
  configureFlags = [
    "ac_cv_prog_HAVELIBUSB=${lib.getDev libusb-compat-0_1}/bin/libusb-config"
  ] ++ lib.optional (!stdenv.hostPlatform.isDarwin) "--with-async-mode";
```

Reply:
I was wondering how to properly specify the path for autoconf. Thank you.

Reply:
IIUC autoconf variables that are computed by some function can be overridden by specifying `ac_cv_<scope>_<variablename>=value` in configureFlags. `<scope>` here happens to be `prog`, but sometimes it's `path`, or `function`, etc... i either just guess until i get it right or grep the autoconf repo ☹️ (e.g. [AC_CHECK_TOOL](https://git.savannah.gnu.org/cgit/autoconf.git/tree/lib/autoconf/programs.m4#n226) is defined to use "prog")